base/CMakeLists.txt

```diff
@@ -69,12 +72,14 @@ set(
   threaded_container.hpp
   threaded_list.hpp
   threaded_priority_queue.hpp
+  internal/message.cpp
```

Comment:
This works just because headers are not build, and this particular header haven't changed for a long time.